docs/en/02_Developer_Guides/01_Templates/03_Requirements.md

```diff
@@ -11,6 +11,68 @@ The examples below are using certain folder naming conventions (CSS files in `cs
 SilverStripe core modules like `cms` use a different naming convention (CSS and JavaScript files in `client/src/`).
 The `Requirements` class can work with arbitrary file paths.
 
+## Exposing static assets
+
+Before requiring a static assets file in PHP code or in a template, those assets need to be "exposed". This process allows SilverStripe projects and SilverStripe modules to make static asset files available via the web server without directly exposing PHP files.
+
+### Configuring your project "exposed" folders
+
+Exposed assets are made available in your web root in a dedicated "resources" directory. Prior to SilverStripe 4.4, the name of this directory was hardcoded to `resources`. In SilverStripe 4.4 and above, the name of the resources directory can be configured by defining the `extra.resources-dir` key in your `composer.json`. SilverStripe projects created from `silverstripe/installer` 4.4 and above will automatically be configured to use `_resources` as their resource directory.
+
+Each folder that needs to be exposed must be entered under the `extra.expose` key in your `composer.json` file. Module developers should use a path relative to the root of their module (don't include the "vendor/package-developer/package-name" path).
+
+This is a sample `composer.json` configured to expose some assets.
```

Comment:
Worth noting that this is a project composer.json file, rather than a module file? It's obvious from the `type` in the example, but maybe it couldn't hurt to add a word or two in this sentence to make it crystal clear?